themes/cloud-native-azure.omp.json

```diff
@@ -0,0 +1,172 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "alignment": "left",
+      "horizontal_offset": 0,
+      "segments": [
+        {
+          "background": "#c386f1",
+          "background_templates": null,
```

Comment:
All null values can be removed (default). Same for empty `powerline_symbol` and others. Could it be I can't see the  icons? They're ideally the `\u34d3` value for example rather than the actual glyph.

Reply:
Any tips on how to convert those? 😰

Reply:
@tomkerkhove we should pair up ;-)